docker/Dockerfile-aci-istio-operator

```diff
@@ -0,0 +1,16 @@
+FROM alpine:3.10.2
+ENV http_proxy=http://proxy.esl.cisco.com:80
+ENV https_proxy=http://proxy.esl.cisco.com:80
```

Comment:
I guess we need to remove this. ;-)

Reply:
@snaiksat I kept it so that my local build wouldn't fail every time. I understand the Jekins would take care of the proxy settings, would it hurt to keep them here? They are reset at end of file. If I remove them, I will have to add these every time I need to build locally.